app/src/auto_validator/core/utils/childhotkey.py

```diff
@@ -8,6 +8,30 @@
 from ..models import Hotkey
 
 
+class CLIManagerWrapper:
+    def __init__(self):
+        self.cli_manager = CLIManager()
+
+    def __enter__(self):
+        sys.stdin = StringIO("y\n")
+        return self
+
+    def __exit__(self, type, value, traceback):
+        sys.stdin = sys.__stdin__
+
+    def stake_revoke_children(self, *args, **kwargs):
+        result = self.cli_manager.stake_revoke_children(*args, **kwargs)
+        return result
```

Comment:
```suggestion
        with self:
            result = self.cli_manager.stake_revoke_children(*args, **kwargs)
        return result
```
and then the caller doesn't need to `with` but can just call a method